src/test/java/com/synopsys/integration/alert/provider/polaris/descriptor/PolarisGlobalDescriptorActionApiTest.java

```diff
@@ -61,8 +62,8 @@ public void validateConfigWhenValidTest() throws Exception {
                                                             .stream()
                                                             .collect(Collectors.toMap(ConfigField::getKey, Function.identity()));
         actionApi.validateConfig(configFieldMap, fieldModel, fieldErrors);
-        assertEquals(ConfigField.REQUIRED_FIELD_MISSING, fieldErrors.get(PolarisDescriptor.KEY_POLARIS_ACCESS_TOKEN));
-        assertEquals(ConfigField.REQUIRED_FIELD_MISSING, fieldErrors.get(PolarisDescriptor.KEY_POLARIS_TIMEOUT));
+        assertNull(fieldErrors.get(PolarisDescriptor.KEY_POLARIS_ACCESS_TOKEN), "Apitoken should be populated with valid token");
```

Comment:
There should be a space between "Api" and "token". We might also want to use "API Token" instead of "Api token".